playbook-website/app/javascript/components/Website/src/layouts/Sidebar/MenuData/GuidelinesNavItems.ts

```diff
@@ -0,0 +1,66 @@
+export const VisualGuidelinesItems = [
```

Comment:
Pascal case here makes the constant appear like a class instead which can be confusing. It might for easier reading if `VISUAL_GUIDELINES_ITEMS=` was used instead. This is a predictable way to declare constants and is clear to other devs.